src/Extensions/SiteTreeContentReview.php

```diff
@@ -333,76 +333,121 @@ public function updateSettingsFields(FieldList $fields)
         }
 
         $module = ModuleLoader::getModule('silverstripe/contentreview');
-        Requirements::javascript($module->getRelativeResourcePath('client/dist/js/contentreview.js'));
+        Requirements::javascript($module->getResource('client/dist/js/contentreview.js')->getRelativePath());
```

Comment:
It may be simpler here to just use `Requirements::javascript('silverstripe/contentreview:client/dist/js/contentreview.js');` and remove the ModuleLoader class import